packages/button/src/Button.tsx

```diff
@@ -341,13 +322,23 @@ export default function Button(props: ButtonProps) {
       >
         {children}
       </span>
-    </button>
+    </Root>
   );
+
+  if (usesCustomElement(props)) {
+    return renderButton(props.as);
+  }
+
+  if (usesLinkElement(props)) {
+    return renderButton('a');
+  }
+
+  return renderButton();
 }
 
 Button.propTypes = {
-  variant: PropTypes.oneOf(['default', 'primary', 'info', 'danger', 'dark']),
-  size: PropTypes.oneOf(['xsmall', 'small', 'normal', 'large']),
+  variant: PropTypes.oneOf(Object.keys(Variant)),
+  size: PropTypes.oneOf(Object.keys(Size)),
```

Comment:
Shouldn't these be `Object.values`, not keys?

Reply:
🤦‍♂ Yep, whoops. Fixed!